scripts/deploy-production.sh

```
@@ -0,0 +1,208 @@
+
+# Deploy to server
+deploy_to_server() {
+    local package_name=$1
+
+    log "Deploying to production server..."
+
+    # Upload package
+    scp "$package_name" "${DEPLOY_USER}@${SERVER_HOST}:/tmp/"
+
+    # Deploy on server
+    ssh "${DEPLOY_USER}@${SERVER_HOST}" << EOF
```

Shellcheck (0.10.0)

[warning] Quote 'EOF' to make here document expansions happen on the server side rather than on the client.

(SC2087)